satpy/tests/reader_tests/test_mviri_l1b_fiduceo_nc.py

```diff
@@ -28,6 +28,7 @@
 import xarray as xr
 from pyproj import CRS
 from pyresample.geometry import AreaDefinition
+from pytest_lazyfixture import lazy_fixture
```

Comment:
This should fix the tests

```suggestion
from pytest_lazy_fixtures import lf as lazy_fixture
```